proxy/src/main/java/org/apache/accumulo/proxy/Proxy.java

```diff
@@ -147,19 +147,16 @@ public void execute(final String[] args) throws Exception {
       final MiniAccumuloCluster accumulo = new MiniAccumuloCluster(folder, "secret");
       accumulo.start();
       clientProps = accumulo.getClientProperties();
-      Runtime.getRuntime().addShutdownHook(new Thread() {
-        @Override
-        public void start() {
-          try {
-            accumulo.stop();
-          } catch (Exception e) {
-            throw new RuntimeException();
-          } finally {
-            if (!folder.delete())
-              log.warn("Unexpected error removing {}", folder);
-          }
+      Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+        try {
+          accumulo.stop();
+        } catch (Exception e) {
+          throw new RuntimeException();
```

Comment:
Should use multi-catch here for checked exceptions, rather than the wide supertype of "Exception", or remove the catch block if there aren't any checked exceptions.

Reply:
Sounds good. Fixed now.